argus/backends/local/local_backend.py

```diff
@@ -0,0 +1,55 @@
+from argus.backends import base as base_backend
+from argus.backends import windows as windows_backend
+from argus.client import windows
+from argus import config as argus_config
+from argus import log as argus_log
+from argus import util
+
+CONFIG = argus_config.CONFIG
+
+
+class LocalBackend(windows_backend.WindowsBackendMixin,
+                   base_backend.BaseBackend,
+                   windows_backend.BaseMetadataProviderMixin):
+    """ Local Backend for testing Windows machines
+        that are already installed and ready"""
+    def __init__(self, name=None, userdata=None, metadata=None,
+                 availability_zone=None):
+        self._name = name
+        self._availability_zone = availability_zone
+        self.userdata = userdata
+        self.metadata = metadata
+        self._username = CONFIG.local.username
+        self._password = CONFIG.local.password
+
+    def get_remote_client(self, username=None, password=None,
```

Comment:
I think you can implement the `WindowsBackendMixin` redefine the `get_remote_client` and user `super` to pass the desired parameters.